packages/jest-jasmine2/src/__tests__/expectation_result_factory.test.js

```diff
@@ -8,7 +8,7 @@
 
 'use strict';
 
-import expectationResultFactory from '../expectation_result_factory';
+import expectationResultFactory from '../expectationResultFactory';
```

Comment:
We also want to change the test names, so `expectation_result_factory.test.js` -> `expectationResultFactory.test.js`. Can you adjust it?